buildbot/configure.py

```diff
@@ -195,6 +200,12 @@ def do_configure(args, passthrough_args):
         "-DLLVM_EXTERNAL_LIBDEVICE_SOURCE_DIR={}".format(libdevice_dir),
         "-DLLVM_EXTERNAL_SYCL_JIT_SOURCE_DIR={}".format(jit_dir),
         "-DLLVM_ENABLE_PROJECTS={}".format(llvm_enable_projects),
+        "-DLLVM_ENABLE_RUNTIMES={}".format(llvm_enable_runtimes),
+        "-DLLVM_LIBC_FULL_BUILD=ON",
+        "-DLLVM_LIBC_ALL_HEADERS=1",
+        "-DLIBC_CONFIG_PATH={}".format(
+            os.path.join(abs_src_dir, "sycl-jit/jit-compiler/lib/libc-config")
+        ),
```

Comment:
I don't really like this, will probably try to have a separate standalone `libc` "build" from inside `sycl-jit`'s CMake. On the other hand, `libc++` doesn't use any fancy options/hack so I plan to leave it under `LLVM_ENABLE_RUNTIMES`.

Any feedback here is very welcome.

Reply:
It specifies what headers and what entry points are implemented for a given target. I'm overriding the "targets" part via `-DLLVM_LIBC_ALL_HEADERS` (partially, because of libc' cmakes) to emit declaration for all entry points, but that only affects user headers generation and not the library build itself (which I'm simply not performing).

By default, on windows only a few things are implemented (user headers + library impl) and it's not enough for me - I need to generate more headers with declaration for much more entrypoints.

Reply:
Particular C library (or POSIX) function/variable/macro/type provided by a given header.